style.css

```diff
@@ -28,7 +28,6 @@
   }
 
   /* ------------ Header text Mobile (header-bg) ------ */
```

Comment:
- [ ]As required for this project The main container should have a `display: grid`, but you didn't use it kindly fix this for your project to match the requirements for this project and your design will look better :wink: